impeller/renderer/backend/vulkan/blit_command_vk.cc

```diff
@@ -262,6 +262,24 @@ bool BlitCopyBufferToTextureCommandVK::Encode(CommandEncoderVK& encoder) const {
                                image_copy               //
   );
 
+  // Transition to shader-read.
+  {
+    BarrierVK barrier;
+    barrier.cmd_buffer = cmd_buffer;
+    barrier.src_access = vk::AccessFlagBits::eColorAttachmentWrite |
```

Comment:
I am not sure why you need the ColorAttachmentWrite src access. Only the transfer write should be sufficient.

Reply:
Done